_opcodes/op206.html

```diff
@@ -16,6 +16,8 @@
 As of EE v2.6, <code>String Reference</code> is bound by
 <details>
 <pre>
+// NOTE: strrefs [0xF00000&mdash;0xF00537] index into <a href="../files/2da/2da_bgee/enginest.htm">enginest.2da</a>, with the row being "strref - 0xF00000"
```

Comment:
This is in preformatted tags, which means the link won't work, but be displayed verbatim.